taskipy/task_runner.py

```diff
@@ -1,13 +1,17 @@
 import sys
-
+import platform
 import psutil  # type: ignore
 import signal
 import subprocess
+import shlex
+import mslex # type: ignore
 from pathlib import Path
-from typing import List, Union
+from typing import List, Union, Optional
 
 from taskipy.pyproject import PyProject
+from taskipy.exceptions import TaskNotFoundError
 from taskipy.task import Task
+from taskipy.help import HelpFormatter
 
 
 class TaskRunner:
```

Comment:
Same question here, whether we want `project` to be "private"?

Reply:
you're right! fixed that :)